src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -255,41 +255,62 @@ function MoneyRequestView({
     );
 
+    let errors = {};
+    if (transaction?.errors) {
+        errors = {
+            ...errors,
+            ...transaction?.errors,
+        };
+    }
+
+    if (parentReportAction?.errors) {
+        errors = {
+            ...errors,
+            ...parentReportAction?.errors,
+        };
+    }
+
     return (
         <View style={[StyleUtils.getReportWelcomeContainerStyle(isSmallScreenWidth, true, shouldShowAnimatedBackground)]}>
             {shouldShowAnimatedBackground && <AnimatedEmptyStateBackground />}
             <View style={shouldShowAnimatedBackground && [StyleUtils.getReportWelcomeTopMarginStyle(isSmallScreenWidth, true)]}>
                 {/* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing */}
-                {(showMapAsImage || hasReceipt) && (
-                    <OfflineWithFeedback
-                        pendingAction={pendingAction}
-                        errors={transaction?.errors}
-                        errorRowStyles={[styles.ml4]}
-                        onClose={() => {
-                            if (!transaction?.transactionID) {
-                                return;
-                            }
-                            Transaction.clearError(transaction.transactionID);
-                        }}
-                    >
-                        <View style={styles.moneyRequestViewImage}>
-                            {showMapAsImage ? (
-                                <ConfirmedRoute transaction={transaction} />
-                            ) : (
-                                <ReportActionItemImage
-                                    thumbnail={receiptURIs?.thumbnail}
-                                    fileExtension={receiptURIs?.fileExtension}
-                                    isThumbnail={receiptURIs?.isThumbnail}
-                                    image={receiptURIs?.image}
-                                    isLocalFile={receiptURIs?.isLocalFile}
-                                    filename={receiptURIs?.filename}
-                                    transaction={transaction}
-                                    enablePreviewModal
-                                />
+                {showMapAsImage ||
+                    hasReceipt ||
+                    (errors && (
+                        <OfflineWithFeedback
+                            pendingAction={pendingAction}
+                            errors={errors}
+                            errorRowStyles={[styles.ml4]}
+                            onClose={() => {
+                                if (!transaction?.transactionID) {
+                                    return;
+                                }
+                                Transaction.clearError(transaction.transactionID);
+                            }}
+                        >
+                            {/* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing */}
```

Comment:
We should not introduce lint errors, kindly fix the error.

Reply:
Lint code /  Run ESLint failed if we don’t put comments on line 292 so we have put here to not blocking any lint same as on line 277

Reply:
If lint is failing we should find the failure and fix it, not override it unless its a very unusual case.

Here its only asking to make null coalescing. Kindly fix it.